README.md

```diff
@@ -1,4 +1,4 @@
-# Azure IoT Edge configuration tool
+# Azure IoT Edge configuration tool [![Build Status](https://dev.azure.com/mseng/VSIoT/_apis/build/status/Azure%20IoT%20Edge/iotedgehubdev?branchName=master)](https://dev.azure.com/Azure-IoT-DDE-EdgeExperience/IoTEdgeConfig/_build?definitionId=28&branchName=main)
```

Comment:
In case you are wondering, this shows a link to the pipeline
![image](https://user-images.githubusercontent.com/11507404/119548867-849cd680-bd4b-11eb-9f62-3f133e5ca748.png)